src/web/App.tsx

```diff
@@ -31,7 +32,11 @@ export function App() {
     else rootRef.current!.classList.remove('darkmode');
   };
 
+  const participantError = useParticipantCheck();
+  if (participantError) return participantError;
```

Comment:
This doesn't seem to work.  When I log in with a user with no participant access, I still get the ugly error message.